scripts/scil_tractogram_math.py

```diff
@@ -113,22 +112,24 @@ def main():
     args = parser.parse_args()
     logging.getLogger().setLevel(logging.getLevelName(args.verbose))
 
+    # Verifications
```

Comment:
Do we really need this comment? the methods are already pretty explicit in what they do...

Reply:
So far, all scripts I checked have been separated into: verifications, loading, processing, saving. Easier to proof-read for me.